<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>860</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>581</width>
     <height>791</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_9">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QVideoWidget" name="Sc1_Video" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>300</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QSlider" name="Sc1_VideoSpeed">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="Sc1_StatusText">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>21</height>
             </size>
            </property>
            <property name="text">
             <string>Media Status</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="Sc1_StatusBin">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>21</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="Sc1_TimeText">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>21</height>
             </size>
            </property>
            <property name="text">
             <string>Running time</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="Sc1_TimeBin">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>21</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="Sc1_Stop">
            <property name="text">
             <string>■</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Sc1_Play">
            <property name="text">
             <string>▶</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Sc1_PlayX2">
            <property name="text">
             <string>▶▶</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Sc1_Pause">
            <property name="text">
             <string>❚❚</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Sc1_Rewind">
            <property name="text">
             <string>◀◀</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="Sc1_Volum">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QLabel" name="Sc1_ListText">
            <property name="maximumSize">
             <size>
              <width>467</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;List&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="Sc1_ListView">
            <row>
             <property name="text">
              <string>1</string>
             </property>
            </row>
            <column>
             <property name="text">
              <string>name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>date</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>length</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QPushButton" name="Sc1_Add">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Sc1_Save">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>Save</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Sc1_Del">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>Del</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Sc1_Result">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>Result</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header location="global">PyQt5.QtMultimediaWidgets</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
